core/src/main/java/hudson/model/Queue.java

```diff
@@ -723,7 +752,7 @@ private synchronized WaitingItem peek() {
         return itemsView.get();
     }
 
-    public synchronized Item getItem(int id) {
```

Comment:
@tfennelly @jglick @stephenc 

This caused [JENKINS-27256](https://issues.jenkins-ci.org/browse/JENKINS-27256), breaking API access to individual queue items (`/queue/item/42/api/xml`), because Stapler does not support methods of the form `getX(long)`, only `getX(int)`.

See:
* http://stapler.kohsuke.org/reference.html *Public Getter Method with an int Argument*
* https://github.com/stapler/stapler/blob/master/core/src/main/java/org/kohsuke/stapler/MetaClass.java#L258...L275